datasets/multinews/multinews.py

```diff
@@ -0,0 +1,153 @@
+import json
+import os
+import datalabs
+from datalabs.tasks import Summarization
+
+_DESCRIPTION = """
+ Multinews dataset for mutlti-document summarization.
+ Each data sample contains multuple documents, which are seperated by "|||||"
+ From paper: "Multi-News: a Large-Scale Multi-Document Summarization Dataset and Abstractive Hierarchical Model" by A. Fabbri et al.
+ See: https://aclanthology.org/P19-1102.pdf
+ See: https://github.com/Alex-Fabbri/Multi-News
+"""
+_CITATION = """\
+    @inproceedings{fabbri-etal-2019-multi,
+    title = "Multi-News: A Large-Scale Multi-Document Summarization Dataset and Abstractive Hierarchical Model",
+    author = "Fabbri, Alexander  and
+      Li, Irene  and
+      She, Tianwei  and
+      Li, Suyi  and
+      Radev, Dragomir",
+    booktitle = "Proceedings of the 57th Annual Meeting of the Association for Computational Linguistics",
+    month = jul,
+    year = "2019",
+    address = "Florence, Italy",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/P19-1102",
+    doi = "10.18653/v1/P19-1102",
+    pages = "1074--1084",
+    abstract = "Automatic generation of summaries from multiple news articles is a valuable tool as the number of online publications grows rapidly. Single document summarization (SDS) systems have benefited from advances in neural encoder-decoder model thanks to the availability of large datasets. However, multi-document summarization (MDS) of news articles has been limited to datasets of a couple of hundred examples. In this paper, we introduce Multi-News, the first large-scale MDS news dataset. Additionally, we propose an end-to-end model which incorporates a traditional extractive summarization model with a standard SDS model and achieves competitive results on MDS datasets. We benchmark several methods on Multi-News and hope that this work will promote advances in summarization in the multi-document setting.",
+}
+"""
+_ABSTRACT = "summary"
+_ARTICLE = "text"
+
+def _gdrive_url(id):
+    return f"https://drive.google.com/uc?id={id}&export=download"
+
+class MultiNewsConfig(datalabs.BuilderConfig):
+    """BuilderConfig for MultiNews."""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for MultiNews.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(MultiNewsConfig, self).__init__(**kwargs)
+
+
+class MultiNewsDataset(datalabs.GeneratorBasedBuilder):
+    """MultiNews Dataset."""
+    BUILDER_CONFIGS = [
+        MultiNewsConfig(
+            name="raw",
+            version=datalabs.Version("1.0.0"),
```

Comment:
Since we have already used the `name` to discriminate different dataset subsets, maybe we can make their Version name the same? (not sure if this is a good idea)

Reply:
version name changed to 1.0.0